apps/backend/src/kernel/mockAnalyzer.service.ts

```diff
@@ -46,33 +50,99 @@ export class MockAnalyzerService extends AbstractCodeAnalyzerService {
         return new Ok({ kind: 'success', value: dependencies });
     }
 
-    validate(sourceCode: string, dependencies: Dependency[]): ValidationResult {
-        return {
-            kind: 'validation_success',
-            success: true
+    private async fetchFile(pkgName: string): Promise<string> {
```

Comment:
The pkg content fetcher is the same as `FilesService.getPublicFile`, but I'm not sure how to make it cleanly common 😕